dotCMS/src/main/java/com/dotmarketing/osgi/GenericBundleActivator.java

```diff
@@ -125,7 +125,13 @@ protected void initializeServices ( BundleContext context ) throws Exception {
 
         this.context = context;
 
-        this.classReloadingStrategy = ClassReloadingStrategy.fromInstalledAgent();
+        try {
+            this.classReloadingStrategy = ClassReloadingStrategy.fromInstalledAgent();
+        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)